specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -3535,6 +3535,125 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/ipAddressAggregates": {
+      "get": {
+        "tags": [
+          "Reports"
+        ],
+        "description": "Gets the ipAddressAggregates for a given service.",
+        "x-ms-examples": {
+          "services_listIpAddressAggregates": {
+            "$ref": "./examples/IpAddressAggregates.json"
+          }
+        },
+        "operationId": "services_listIpAddressAggregates",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "skiptoken",
+            "in": "query",
+            "description": "Used to paginate results",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The list of ip addresses.",
+            "schema": {
+              "$ref": "#/definitions/IpAddressAggregates"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/ipAddressAggregateSettings": {
+      "get": {
+        "tags": [
+          "Reports"
+        ],
+        "description": "Gets the ipAddressAggregateSettings.",
+        "x-ms-examples": {
+          "services_getIpAddressAggregateSettings": {
+            "$ref": "./examples/IpAddressAggregateSettings.json"
+          }
+        },
+        "operationId": "services_getIpAddressAggregateSettings",
```

Comment:
Same as above. Verb_Noun format.

Reply:
Fixed, changed to get_ipAddressAggregateSettings